html-relative-links/projects/projects.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Projects</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href = "../index.html">Home Page</a></li>
+    <li><a href = "../about-me/about-me.html">About Me</a></li>
+    <li><a href = "../projects/projects.html">Projects</a></li>
```

Comment:
Since we're already in the `projects` folder, we can just do `<a href="projects.html">`